internal/broker/broker_test.go

```diff
@@ -661,6 +669,50 @@ func TestEndSession(t *testing.T) {
 	require.NoError(t, err, "EndSession should not have returned an error when ending an existent session")
 }
 
+func TestUserPreCheck(t *testing.T) {
+	t.Parallel()
+
+	tests := map[string]struct {
+		username        string
+		allowedSuffixes []string
+
+		wantErr bool
+	}{
+		"Successfully allow user": {},
```

Comment:
I find the default case here a little bit confusing as we don’t have many of them, and I would expect the default config to be the default shipped to the user, so no-one permitted.

We don’t have yet a lot of defaults using the default username, maybe we should just skip the "empty" trick, wdyt?